src/components/autocomplete/autocomplete.scss

```diff
@@ -0,0 +1,161 @@
+  }
+  50% {
+    opacity: 0;
+    height: 40px;
+    animation-timing-function: ease-in;
+  }
+  100% {
+    height: 0;
+    opacity: 0;
+  }
+}
+@keyframes list-in {
+  0% {
+    opacity: 0;
+    height: 0;
+    animation-timing-function: linear;
+  }
+  50% {
+    opacity: 1;
+    height: 40px;
+    animation-timing-function: ease-out;
+  }
+  100% {
+    opacity: 1;
+    height: 40px;
+  }
+}
+body {
+  max-width: 800px;
+  margin: 0 auto;
+  height: 100%;
+  md-content {
+    overflow: visible;
+  }
+  md-autocomplete {
+    background: white;
+    box-shadow: 0 2px 5px rgba(black, 0.25);
+    border-radius: 2px;
+    display: block;
+    height: 40px;
+    position: relative;
+    overflow: visible;
+
+    md-autocomplete-wrap {
+      display: block;
+      position: relative;
+      overflow: visible;
+      height: 40px;
+
+      md-progress-linear {
+        position: absolute;
+        bottom: 0; left: 0; width: 100%;
+        height: 3px;
+        transition: none;
+
+        .md-container {
+          transition: none;
+          top: auto;
+          height: 3px;
+        }
+        &.ng-enter {
+          transition: opacity 0.15s linear;
+          &.ng-enter-active {
+            opacity: 1;
+          }
+        }
+        &.ng-leave {
+          transition: opacity 0.15s linear;
+          &.ng-leave-active {
+            opacity: 0;
+          }
+        }
+      }
+    }
+    input {
+      position: absolute;
+      left: 0;
+      top: 0;
+      width: 100%;
+      box-sizing: border-box;
+      border: none;
+      box-shadow: none;
+      padding: 0 15px;
+      font: 400 14px/40px Roboto, sans-serif;
+      height: 40px;
+      outline: none;
+      z-index: 2;
+      background: transparent;
+    }
+    button {
+      position: absolute;
+      top: 10px;
+      right: 10px;
+      line-height: 20px;
+      z-index: 2;
+      text-align: center;
+      width: 20px;
+      height: 20px;
+      cursor: pointer;
+      border: none;
+      background: #999;
+      color: white;
+      border-radius: 50%;
+      padding: 0;
+      font-size: 12px;
+      &.ng-enter {
+        transform: scale(0);
+        transition: transform 0.15s ease-out;
+        &.ng-enter-active {
+          transform: scale(1);
+        }
+      }
+      &.ng-leave {
+        transition: transform 0.15s ease-out;
+        &.ng-leave-active {
+          transform: scale(0);
+        }
+      }
+    }
+    ul {
+      position: absolute;
+      top: 100%;
+      left: 0;
+      right: 0;
+      background: white;
+      box-shadow: 0 2px 5px rgba(black, 0.25);
+      margin: 0;
+      list-style: none;
+      padding: 0;
+      overflow: auto;
+      max-height: 41px * 5.5;
+      li {
+        border-top: 1px solid #ddd;
+        padding: 0 15px;
+        font: 400 14px/40px Roboto, sans-serif;
+        overflow: hidden;
+        height: 40px;
+        transition: background 0.15s linear;
+        cursor: pointer;
```

Comment:
This (and `:hover`) would normally be cause for concern, since `li` is not reachable from the keyboard by default. But because the arrow keys allow you to reach list items from the input, it works in this case.